server/views/pages/update-goal.njk

```diff
@@ -113,52 +113,52 @@
                         </tbody>
                     </table>
                 {% else %}
-                    <div class="goal-summary-card__steps--empty">
-                        <strong>{{ locale.stepsList.noStepsAdded }}</strong>
-                        {% if data.actions.length > 0 %}
-                            <a href="/goal/{{ data.goal.uuid }}/add-steps">{{ locale.stepsList.addSteps }}</a>
-                        {% endif %}
+                    <div class="govuk-summary-card govuk-summary-card--no-pad">
```

Comment:
The markup here is different to that in goal-summary-card which displays when there are no steps - can you compare and consolidate to one approach please?

Reply:
Updated in https://github.com/ministryofjustice/hmpps-sentence-plan-ui/pull/186/commits/2a84f8fc79e24634664948ecb231cd94aa9de143